tests/integration/test_logcollector/test_only_future_events/test_only_future_events.py

```diff
@@ -148,27 +141,30 @@ def test_only_future_events(configure_local_internal_options_module, get_configu
     tier: 0
 
     parameters:
-        - get_local_internal_options:
-            type: fixture
-            brief: Get local internal options from the module.
-        - configure_local_internal_options:
+        - configuration:
```

Comment:
Add tests_phases documentation, like https://github.com/wazuh/wazuh-qa/blob/4.4/tests/integration/test_analysisd/test_limit_eps/test_event_processing.py#L85-L100

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3484/commits/c381044fd71b864462fea0dc8549ca5865dac7bd)